### ISSUE TITLE
Awaited task created by `asyncio.create_task()` labeled as not accessed.

### ISSUE THREAD

**Issue by pawelrubin** [microsoft/pyright#775]
**Describe the bug**
Awaited task created by `asyncio.create_task()` labeled as _not accessed._

**To Reproduce**
Steps to reproduce the behavior.

Consider this piece of code:
```python
import asyncio


async def foo() -> None:
    while True:
        await asyncio.sleep(1)
        print("still working")


async def main():
    foo_task = asyncio.create_task(foo())
    await foo_task


asyncio.run(main())
```

Hovering over `foo_task` produces the following error: `"foo_task" is not accessed` 

**Expected behavior**
There should be no errors.

**Additional context**
- VS Code extension: - 1.1.47
  - settings:
    ```json
		{
  			"pyright.typeCheckingMode": "strict",
  			"pyright.useLibraryCodeForTypes": true
		}
	```
- Python: 3.8.3

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of Pyright.

**Comment by erictraut** [comment]
This is now fixed in Pyright version 1.1.48, which I just published. It will also be included in the next public release of Pylance.
